<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>import_images_tool</class>
 <widget class="QDialog" name="import_images_tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Sequence</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>276</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="startimgBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="numimgBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="startimgButton">
        <property name="text">
         <string>Start Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="numimgButton">
        <property name="text">
         <string># files in the same folder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="incButton">
        <property name="text">
         <string>Increment</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="incBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="filterButton">
        <property name="text">
         <string>File Name Contains</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="filterEdit"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QComboBox" name="comboPack"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="startimgButton_2">
        <property name="text">
         <string>End Image</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="endimgBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>info: the Names of the filtered images 
are sorted numerically.</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>import_images_tool</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>import_images_tool</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
